packages/daemon/DESIGN.md

```diff
@@ -0,0 +1,90 @@
+
+Endo is a user agent for running guest programs.
+These programs receive capabilities from the user in the form of references to
+potentially powerful remote objects.
+To that end, Endo exhibits a system called Eventual Send that stretches
```

Comment:
"exhibits" is an odd choice. I might have said "provide". But no change suggested.